tests/qnn/test_keras.py

```diff
@@ -979,6 +1002,7 @@ def circuit(inputs, w1, w2):
     assert info["device_name"] == "default.qubit"
 
 
+@requires_keras2
```

Comment:
Is this the only place we use this decorator? If so, does it make sense to keep it if we only use it once?

Reply:
This is the weirdest phenomenon I observed these days. Actually, in my local tests 1/3 of the test should fail. However, in our CI eventually only this test fails.
Since `KerasLayer` will be deprecated soon and removed in 0.42, I'll say totally fine to get rid of the decorator.

Reply:
https://github.com/PennyLaneAI/pennylane/pull/7039/commits/c4b8c6c352a4b60d48a7074728eeed00dcf3d384